src/pages-and-resources/discussions/app-config-form/apps/lti/LtiConfigForm.jsx

```diff
@@ -27,6 +28,10 @@ function LtiConfigForm({
     onSubmit,
   });
 
+  const isInvalidConsumerKey = touched.consumerKey && errors.consumerKey;
```

Comment:
Turns out that without checking `touched`, the messages became invalid as soon as they were loaded.  Oops.